tests/testChecker.ml

```diff
@@ -325,6 +464,93 @@ let suite =
        assert_operation_list_equal ~expected:expected_ops ~real:ops
     );
 
+    ("entrypoint_liquidation_auction_place_bid - emits expected operations" >::
+     fun _ ->
+       Ligo.Tezos.reset ();
+       let checker = checker_with_active_auction () in
+       (* Lookup the current minimum bid *)
+       let auction_details = Checker.view_current_liquidation_auction_minimum_bid ((), checker) in
+       (* Mint some kit to be able to bid *)
+       let new_burrow_no = Ligo.nat_from_literal "100n" in
+       Ligo.Tezos.new_transaction ~seconds_passed:0 ~blocks_passed:0 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "1_000_000_000mutez");
+       let _, checker = Checker.entrypoint_create_burrow (checker, (new_burrow_no, None)) in
+       Ligo.Tezos.new_transaction ~seconds_passed:0 ~blocks_passed:0 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let _, checker = Checker.entrypoint_mint_kit (checker, (new_burrow_no, auction_details.minimum_bid)) in
+
+       (* Place a bid *)
+       Ligo.Tezos.new_transaction ~seconds_passed:10 ~blocks_passed:1 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let _ops, _checker = Checker.entrypoint_liquidation_auction_place_bid
+           (checker,
+            ((Option.get checker.liquidation_auctions.current_auction).contents, auction_details.minimum_bid))
+       in
+       ()
```

Comment:
Instead of ignoring the `ops` here we could check that they are indeed empty, like so:
```suggestion
       let ops, _checker = Checker.entrypoint_liquidation_auction_place_bid
           (checker,
            ((Option.get checker.liquidation_auctions.current_auction).contents, auction_details.minimum_bid))
       in
       assert_operation_list_equal ~expected:[] ~real:ops
```

Reply:
Also, since we've come this far (this applies to the tests that follow as well), we might as well check that the whole effect of the operation called is as expected (i.e., inspect the resulting `checker` too, and for example in this case check that the current winning bid is the one we placed).

Reply:
Actually, now that I think about it, this pattern of testing only parts of the output shows up in several places in our testsuite. We can consider changing this, but it's not part of this PR. I'm resolving this one.